hack/demo/deploy/app_config.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: core.oam.dev/v1alpha2
```

Comment:
Why don't reference https://github.com/oam-dev/samples/blob/1.0.0-alpha2/2.ServiceTracker_App/ApplicationConfiguration/tracker-app-config-managed.yaml?

So when the YAML is updated there, you don't need to sync.

Reply:
`port` is different.
Need to make port configurable in [samples](https://github.com/oam-dev/samples/blob/1.0.0-alpha2/2.ServiceTracker_App/ApplicationConfiguration/tracker-app-config-managed.yaml).
Just want to make this thing usable now.